tests/adapter/dbt/tests/adapter/simple_seed/test_seed.py

```diff
@@ -229,32 +234,24 @@ class TestSimpleSeedWithBOM(SeedConfigBase):
     @pytest.fixture(scope="class", autouse=True)
     def setUp(self, project):
         """Create table for ensuring seeds and models used in tests build correctly"""
-        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+        project.run_sql(seeds__expected_sql)
 
-        # manual copy because seed has a special and tricky-to-include unicode character at 0
```

Comment:
do we need to preserve these as actual `.csv` files to validate this? I'm wondering if it would be easier to have a core-only test that looks at some of these edge cases (BOM, unicode etc.) and an actually "simple" seed test in the adapter zone that just validates loading csv files

Reply:
I think that would be a better approach. I'll modify the code.